src/sentry/static/sentry/app/views/organizationEventsV2/eventDetails.jsx

```diff
@@ -99,6 +99,14 @@ class EventDetails extends AsyncComponent {
     throw new Error('Could not determine projectId');
   }
 
+  componentDidMount() {
+    document.body.style.overflow = 'hidden';
```

Comment:
Unless you're planning on opening modals within modals, this may work fine. 

Depending on browsers you want to support, I generally I some success with this library: https://github.com/willmcpo/body-scroll-lock

Reply:
Will this cause issues with body's current CSS for `overflow`? (it sets `overflow-x` as `hidden`)

Reply:
Adding and removing a class might be less likely to interfere with custom styles on another page

Reply:
I could store the current overflow setting so that we can restore the previous state back on unmount.

Reply:
> Unless you're planning on opening modals within modals, this may work fine.

Not planning on doing that with this modal.